bundles/org.eclipse.e4.ui.workbench.renderers.swt/META-INF/MANIFEST.MF

```diff
@@ -27,7 +27,8 @@ Require-Bundle: org.eclipse.e4.ui.workbench;bundle-version="0.9.0",
  org.eclipse.e4.ui.css.swt;bundle-version="0.11.0",
  org.eclipse.e4.core.di.extensions;bundle-version="0.12.0",
  org.eclipse.equinox.preferences;bundle-version="3.5.200",
- org.eclipse.core.runtime;bundle-version="3.17.100"
+ org.eclipse.core.runtime;bundle-version="3.17.100",
+ org.eclipse.ui;bundle-version="[3.201.0,4.0.0)"
```

Comment:
You can't add `org.eclipse.ui` dependency here

Reply:
removed